www/docs/getting-started/client.md

```diff
@@ -266,7 +266,7 @@ You can also set these parameters through [`provider.authorizationParams`](/conf
 :::
 
 :::note
-The following parameters are always overridden server-side: `redirect_uri`, `scope`, `state`
+The following parameters are always overridden server-side: `redirect_uri`, `state`
```

Comment:
not sure abt `state` tbh

Reply:
it's set in a different place, so the note is correct now. 😊